core/shared/src/main/scala/io/udash/properties/seq/SeqPropertyFromSingleValue.scala

```diff
@@ -7,7 +7,7 @@ import io.udash.utils.{Registration, SetRegistration}
 import scala.collection.mutable
 
 private[properties]
-abstract class BaseReadableSeqPropertyFromSingleValue[A, B: PropertyCreator](
+abstract class BaseReadableSeqPropertyFromSingleValue[A, B: PropertyCreator : DefaultValue](
```

Comment:
I guess this is the reason why `DefaultValue` was needed above. Is it really necessary here? We shouldn't need `DefaultValue` once the `Property` is created.